### ISSUE TITLE
[a11y] Cannot create a loan from a list of catalog numbers

### ISSUE THREAD

**Issue by grantfitzsimmons** [specify/specify7#1400]
**Describe the bug**
Cannot create a loan from a list of catalog numbers

**To Reproduce**
1. Create a loan
2. Enter Cat # 's
3. Click next
4. see nothing

**Screenshots**
![image](https://user-images.githubusercontent.com/37256050/167145050-da1b4600-5d3e-4aaf-95cd-89263ec10096.png)

Windows 10, Edge

**Comment by maxxxxxdlp** [comment]
Fixed in 706d3ba0
